lib/service/database.js

```diff
@@ -336,10 +325,11 @@ DatabaseService.prototype.queueAnalysisOperations = function(analysis, callback)
                     query: updateNodeStatusAtAnalysisCatalogQuery(nodeIds, Node.STATUS.RUNNING)
                 });
                 asyncQueries.push({
+                    id: [analysis.id(), nodeToUpdate.id(), nodeToUpdate.getType()].join(':'),
```

Comment:
For instance:

`8563bc5307ba5f7231bbc35f4bffb4173cd7f693:8563bc5307ba5f7231bbc35f4bffb4173cd7f693:sampling`